include/openssl/provider.h

```diff
@@ -44,6 +44,27 @@ int OSSL_PROVIDER_get_capabilities(const OSSL_PROVIDER *prov,
                                    OSSL_CALLBACK *cb,
                                    void *arg);
 
+/*-
+ * Provider configuration parameters are normally set in the configuration file,
+ * but can also be set early in the main program before a provider is in use by
+ * multiple threads.  The "default" provider does not load any parameters from
+ * the configuration file and this is the only way to set "default" provider
+ * parameters.
```

Comment:
Again, this last sentence is not strictly true. Parameters will always be loaded from the config file. Jut not necessarily looked at by the provider.